### DISCUSSION TITLE
How to get the point at which the variable equals to a given value

### DISCUSSION THREAD

**Discussion by xuxiaobei1995** [idaholab/moose#17924] [Q&A General] [answered]
Hi all,

In my code I want to get the position of the point at which the variable equals to a given value. Take a 1-D case for example:
![获得给定值对应点](https://user-images.githubusercontent.com/34068460/119529490-bdbd6100-bdb4-11eb-8e45-8fa81587ba56.PNG)
The task is to get the point xm at which the variable equals to the given value m. Does any one know the way to implement this using a MooseObject (such as the Postprocessor)?  Any suggestion is appreciated.

Xiaobei

**Comment by dschwen** [comment, marked as answer]
Take a look at `FindValueOnLine`

**Reply by xuxiaobei1995** [reply]
Thank you Daniel! This is just what I need.
